django/contrib/admin/options.py

```diff
@@ -1556,11 +1598,13 @@ def response_post_save_change(self, request, obj):
         """
         return self._response_post_save(request, obj)
 
-    def response_action(self, request, queryset):
+    def response_action(self, request, queryset, change):
```

Comment:
Now `queryset` param could be an object. Should we rename it? Maybe "selected"? 🤔